packages/fxa-payments-server/package.json

```diff
@@ -15,12 +15,15 @@
     "build": "tsc --build ../fxa-react && NODE_ENV=production npm run build-css && SKIP_PREFLIGHT_CHECK=true PUBLIC_URL=/ INLINE_RUNTIME_CHUNK=false CI=false rescripts build",
```

Comment:
I believe we want to run `grunt merge-ftl` on `start` similar to settings; having just the watch command running doesn't run it the command right away on `yarn start` so if strings have changed between `yarn stop` and `yarn start` it doesn't update. (I don't think we need it in `postinstall` like it is in settings though, since it gets overwritten, that's probably something we can remove on the settings side)